### ISSUE TITLE
Adguard Addon not starting due to invalid Config File

### ISSUE THREAD

**Issue by plvoy-de** [hassio-addons/addon-adguard-home#184]
```
-----------------------------------------------------------
 Add-on: AdGuard Home
 Network-wide ads & trackers blocking DNS server
-----------------------------------------------------------
 Add-on version: 4.1.2
 You are running the latest version of this add-on.
 System: Home Assistant OS 5.13  (amd64 / qemux86-64)
 Home Assistant Core: 2021.5.5
 Home Assistant Supervisor: 2021.04.3
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to DEBUG
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] adguard.sh: executing... 
[11:28:18] DEBUG: Requested API resource: http://supervisor/network/interface/default/info
[11:28:18] DEBUG: Request method: GET
[11:28:18] DEBUG: Request data: {}
[11:28:18] DEBUG: API HTTP Response code: 200
[11:28:18] DEBUG: API Response: {"result": "ok", "data": {"interface": "enp2s1", "type": "ethernet", "enabled": true, "connected": true, "primary": true, "ipv4": {"method": "static", "address": ["(redacted-ipv4)/23"], "nameservers": ["(redacted-ipv4)"], "gateway": "(redacted-ipv4)"}, "ipv6": {"method": "auto", "address": ["(redacted-ipv6)/64", "(redacted-ipv6)/128", "(redacted-ipv6)/64"], "nameservers": ["(redacted-ipv6)"], "gateway": "(redacted-ipv6)"}, "wifi": null, "vlan": null}}
[11:28:18] INFO: Adding (redacted-ipv4)
[11:28:18] INFO: Adding (redacted-ipv6)/64
(redacted-ipv6)/128
(redacted-ipv6)
[11:28:18] INFO: Adding 172.30.32.1
[cont-init.d] adguard.sh: exited 0.
[cont-init.d] discovery.sh: executing... 
[11:28:18] DEBUG: Requested API resource: http://supervisor/discovery
[11:28:18] DEBUG: Request method: POST
[11:28:18] DEBUG: Request data: {"service":"adguard","config":{"host":"127.0.0.1","port":45158}}
[11:28:18] DEBUG: API HTTP Response code: 200
[11:28:18] DEBUG: API Response: {"result": "ok", "data": {"uuid": "dc920227b81742199b24d3b130a24660"}}
[11:28:18] DEBUG: Filtering response using: .uuid
[11:28:18] INFO: Successfully send discovery information to Home Assistant.
[cont-init.d] discovery.sh: exited 0.
[cont-init.d] nginx.sh: executing... 
[11:28:18] DEBUG: Requested API resource: http://supervisor/addons/self/info
[11:28:18] DEBUG: Request method: GET
[11:28:18] DEBUG: Request data: {}
[11:28:18] DEBUG: API HTTP Response code: 200
[11:28:18] DEBUG: API Response: {"result": "ok", "data": {"name": "AdGuard Home", "slug": "a0d7b954_adguard", "hostname": "a0d7b954-adguard", "dns": ["a0d7b954-adguard.local.hass.io"], "description": "Network-wide ads & trackers blocking DNS server", "long_description": "# Home Assistant Community Add-on: AdGuard Home
[![Release][release-shield]][release] ![Project Stage][project-stage-shield] ![Project Maintenance][maintenance-shield]
[![Discord][discord-shield]][discord] [![Community Forum][forum-shield]][forum]
[![Sponsor Frenck via GitHub Sponsors][github-sponsors-shield]][github-sponsors]
[![Support Frenck on Patreon][patreon-shield]][patreon]
Network-wide ads & trackers blocking DNS server.
## About
AdGuard Home is a network-wide ad-and-tracker blocking DNS server with
parental control (adult content blocking) capabilities. Its purpose is to let
you control your entire network and all your devices, and it does not require
using a client-side program.
AdGuard Home provides a beautiful, easy and feature-rich web interface to
easily manage the filtering process and its settings.
![AdGuard Home in the Home Assistant frontend][screenshot]
[discord-shield]: https://img.shields.io/discord/478094546522079232.svg
[discord]: https://discord.me/hassioaddons
[forum-shield]: https://img.shields.io/badge/community-forum-brightgreen.svg
[forum]: https://community.home-assistant.io/t/home-assistant-community-add-on-adguard-home/90684?u=frenck
[github-sponsors-shield]: https://frenck.dev/wp-content/uploads/2019/12/github_sponsor.png
[github-sponsors]: https://github.com/sponsors/frenck
[maintenance-shield]: https://img.shields.io/maintenance/yes/2021.svg
[patreon-shield]: https://frenck.dev/wp-content/uploads/2019/12/patreon.png
[patreon]: https://www.patreon.com/frenck
[project-stage-shield]: https://img.shields.io/badge/project%20stage-production%20ready-brightgreen.svg
[release-shield]: https://img.shields.io/badge/version-v4.1.2-blue.svg
[release]: https://github.com/hassio-addons/addon-adguard-home/tree/v4.1.2
[screenshot]: https://github.com/hassio-addons/addon-adguard-home/raw/v4.1.2/images/screenshot.png", "advanced": false, "stage": "stable", "auto_update": true, "repository": "a0d7b954", "version": "4.1.2", "version_latest": "4.1.2", "update_available": false, "protected": true, "rating": 5, "boot": "auto", "options": {"ssl": false, "certfile": "fullchain.pem", "keyfile": "privkey.pem", "log_level": "debug"}, "schema": [{"name": "log_level", "optional": true, "type": "select", "options": ["trace", "debug", "info", "notice", "warning", "error", "fatal"]}, {"name": "ssl", "required": true, "type": "boolean"}, {"name": "certfile", "required": true, "type": "string"}, {"name": "keyfile", "required": true, "type": "string"}, {"name": "leave_front_door_open", "optional": true, "type": "boolean"}], "arch": ["aarch64", "amd64", "armhf", "armv7", "i386"], "machine": [], "homeassistant": "0.113.2", "url": "https://github.com/hassio-addons/addon-adguard-home", "state": "started", "detached": false, "available": true, "build": false, "network": {"53/udp": 53, "80/tcp": null}, "network_description": {"53/udp": "DNS server port", "80/tcp": "Web interface (Not required for Ingress)"}, "host_network": true, "host_pid": false, "host_ipc": false, "host_dbus": false, "privileged": [], "full_access": false, "apparmor": "default", "devices": [], "icon": true, "logo": true, "changelog": true, "documentation": true, "stdin": false, "webui": null, "hassio_api": true, "hassio_role": "manager", "auth_api": true, "homeassistant_api": false, "gpio": false, "usb": false, "uart": false, "kernel_modules": false, "devicetree": false, "udev": false, "docker_api": false, "video": false, "audio": false, "audio_input": null, "audio_output": null, "startup": "services", "services": [], "discovery": ["adguard"], "ip_address": "172.30.32.1", "translations": {}, "ingress": true, "ingress_entry": "/api/hassio_ingress/becRUHw3KPB8yaLL5C9S-CFwU0Ehqh3-r6Bcz-_KXyQ", "ingress_url": "/api/hassio_ingress/becRUHw3KPB8yaLL5C9S-CFwU0Ehqh3-r6Bcz-_KXyQ/", "ingress_port": 63153, "ingress_panel": true, "watchdog": true}}
[cont-init.d] nginx.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[11:28:18] INFO: Starting AdGuard Home server...
[services.d] done.
2021/05/20 11:28:18.908757 [info] AdGuard Home, version v0.106.3
2021/05/20 11:28:18.909958 [error] Couldn't parse config file: invalid IP address: (redacted-ipv6)/64
(redacted-2nd-ipv6)
```

I cant find any information on where to find the config file, if any exists outside the container. I could ofcourse Uninstall/Reinstall, but maybe someone else runs into this problem as well.

**Comment by frenck** [comment]
This is actually fixed in 4.1.3, please upgrade.

**Comment by plvoy-de** [comment]
yes, works again, thank you!
